twinkle.js

```diff
@@ -482,6 +482,10 @@ Twinkle.load = function () {
 		mw.util.addCSS('.morebits-dialog-content, .morebits-dialog-footerlinks { font-size: 100% !important; } ' +
 			'.morebits-dialog input, .morebits-dialog select, .morebits-dialog-content button { font-size: inherit !important; }');
 	}
+
+	if ($('#p-twinkle').length === 0) {
```

Comment:
This should work I suppose, but this is only an issue for vector menu users, right?  I don't think having unnecessary `initial` would muck anything up, but is there a reason not to be specific?

```suggestion
	if (mw.config.get('skin') === 'vector' && Twinkle.getPref('portletType') === 'menu' && $('#p-twinkle').length === 0) {
```

Reply:
I would suppose most anything touching this for non-vectormenu systems to be loaded after Twinkle, but given this whole thing is a result of not wanting the menu to jump, I guess it's more than possible if not entirely likely to interact.  Still, if something's touching this in non-vector, it's probably also touching it in vector, at which point there's been a collision and they've probably fixed it.